mount/src/assets/comment.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import SortIcon from '@mui/icons-material/Sort';
+
+
+export default class Posts extends React.Component {
+  render() {
+    const comments = this.props.comments;
```

Comment:
don't create variable in render , as this.props.comments will again and again assign into  comments variable,
directly use this.props.comments in place of comments variable